platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -149,7 +149,7 @@ extension PolyOut {
                 completionHandler(cachedEntries, nil)
                 return
             }
-            let targetUrl = pathFromId(id: url)
+            let targetUrl = filesystemUrlFromId(id: url)
```

Comment:
The fact that you're using `url` as an `id` further supports the fact that it's an URI, not an URL.

Reply:
Yap, I can use uri.